src/LinSys/linsysmerger.ci

```diff
@@ -200,7 +200,7 @@ module linsysmerger {
       entry void wait4lhs() {
         when lhsbc_complete() serial "hyprelhs" { hyprelhs(); } };
       entry void wait4rhs() {
-        when rhsbc_complete() serial "hyprerhs" { hyprerhs(); } };
+        when rhsbc_complete_rhs() serial "hyprerhs" { hyprerhs(); } };
```

Comment:
Ah! I think there is something we are missing with the way we understand multiple use of SDAG triggers/messages.

I was just able to produce an example, that does _not_ work if you use unique names for the triggers, but works with using the same names.

@rspavel: can you please verify that you can reproduce the following?

Check out 715b8a82fe3081bcdbea64a036be0be25ecc16db, compile and run
```
ctest -j36 -R asynclogic_u0.5_pe8 -V
```
It should run fine.

Then check out a156faa5cc60ba26354a28a96055026b39bd46b4, compile and run the same, which hands for me.

The only difference is that a156faa5cc60ba26354a28a96055026b39bd46b4 uses uniquely named SDAG triggers.

Note that the logic requires `lhs_complete()` _and_ the completion of LinSysMerger::addbc(). I believe both versions should work, but for some reason only the version using the _non-uniquely-named_ SDAG triggers works.

This is beyond my understanding at this point...

I have no illusions that the asynchronous SDAG logic in LinSysMerger is correct, because the regression test suite fails with a randomized-queue-build of Charm++. That is why we need to get #131 working and in the meantime not screw up much.

Because of this, out of caution, I am reverting this to not-approved. Sorry, I really liked the uniquely-named triggers...